akamaiwaf_connector.py

```diff
@@ -609,6 +625,35 @@ def _handle_activation_details(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_siteshields(self, param):
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # make rest call
+        endpoint = "../../siteshield/v1/maps"
+        ret_val, response = self._make_rest_call(endpoint, action_result, params=None, headers=None)
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        siteShieldsData = response
```

Comment:
suggestion:
```python
ret_val, siteShieldsData = self._make_rest_call(endpoint, action_result, params=None, headers=None)
if phantom.is_fail(ret_val):
  return action_result.get_status()
```

Reply:
Hello @siisar, Could you please address the suggested change? Thank you!

Reply:
ok, feel free to adapt those minor changes to your own style, I used response var as a standar for the return values of self._make_rest_call, but feel free to make any changes

Regards

Reply:
Removed unused variable, and used snake_case instead of camelCase as per python best practices.